src/user/projects/projects.scss

```diff
@@ -84,4 +68,20 @@
 
     }
     margin-left: 10%;
-  }
\ No newline at end of file
+  }
+
+  @media (min-width: 320px) and (max-width: 1024px){
+    .organization {
+      margin-left: 0;
+      .projects {
+        margin: 0;
+        #projects_header {
+          text-align: center;
+        }
```

Comment:
If possible avoid using Id selectors

Reply:
It was already used I just wrapped under media queries.